infrastructure/logging/build.gradle

```diff
@@ -2,4 +2,6 @@ dependencies {
     api 'org.apache.tuweni:tuweni-bytes'
 
     implementation 'org.apache.logging.log4j:log4j-core'
+
+    testImplementation testFixtures(project(':infrastructure:io'))
```

Comment:
I'm a bit worried this creates a cyclic dependency as everything tends to depend on logging's testFixtures.  Gradle and IntelliJ usually handle it but it breaks eclipse.  I think we generally need to move to the policy that you shouldn't  use testFixtures from a module you aren't using as an actual dependency.

On the plus side, I'm not seeing where it's actually used in the test so hopefully it can just be removed.

Reply:
Oh I see...
The test was using `OSUtils` from there, but it is not a big deal to adjust it to not use this class: c465f53